sdk/python/test/e2e/test_e2e_pytorchjob.py

```diff
@@ -240,6 +249,79 @@ def test_sdk_e2e_create_from_image(job_namespace):
     TRAINING_CLIENT.delete_job(JOB_NAME, job_namespace)
 
 
+@pytest.mark.skipif(
+    GANG_SCHEDULER_NAME in GANG_SCHEDULERS,
+    reason="For plain scheduling",
+)
+def test_sdk_e2e_create_from_train_api(job_namespace):
+    JOB_NAME = "pytorchjob-from-train-api"
+
+    num_workers = 1
```

Comment:
```suggestion
```

It seems that this refers to only one.

Reply:
Yes. Thank you for pointing it out! I will update accordingly.